<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-72HVRCC" failures="1" tests="5" name="Test" time="37.213" errors="0" timestamp="2025-04-15T23:21:21 IST">
  <testcase classname="testCases.TC001_AccountRegistrationTest" name="verify_Account_Registration" time="2.158"/>
  <testcase classname="testCases.TC003_loginDDT" name="verify_loginDDT" time="2.459"/>
  <testcase classname="testCases.TC003_loginDDT" name="verify_loginDDT" time="1.473">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_loginDDT.verify_loginDDT(TC003_loginDDT.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_loginDDT" name="verify_loginDDT" time="11.282"/>
  <testcase classname="testCases.TC003_loginDDT" name="verify_loginDDT" time="11.577"/>
</testsuite> <!-- Test -->
